com.github.rizinorg.Cutter.yml

```diff
@@ -0,0 +1,29 @@
+#flatpak-builder --force-clean --user --install ./build ./com.github.rizinorg.cutter.yml
+app-id: com.github.rizinorg.Cutter
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: cutter
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --filesystem=home
+  - --device=dri
+  - --allow=devel
+modules:
+  - name: cutter
+    buildsystem: cmake
+    builddir: true
+    build-options:
+      build-args:
+        - --share=network
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DCUTTER_ENABLE_KSYNTAXHIGHLIGHTING=on
+#      - -DCUTTER_ENABLE_PYTHON=on
+#      - -DCUTTER_ENABLE_PYTHON_BINDINGS=on
+#      - -DCUTTER_ENABLE_GRAPHVIZ=on
+    sources:
+      - type: git
+        url: https://github.com/rizinorg/cutter
```

Comment:
you must specify a tag or a revision for git.

Reply:
It was temporary to  test a first build. I've updated it.